src/material/standard/shader/metallic-roughness.frag

```diff
@@ -363,7 +363,7 @@ void main()
 #endif
 
 #ifdef MATERIAL_UNLIT
-    FRAG_COLOR = vec4(LINEARtoSRGB(baseColor.rgb) * baseColor.a, baseColor.a);
+    FRAG_COLOR = vec4(LINEARtoSRGB(baseColor.rgb), baseColor.a);
```

Comment:
premultiplied alpha is decided in the material state.blendmode

as state.blendMode is set to BLEND_MODES.NORMAL by default, it will be premultiplied

if we were to set state.blendmode to BLEND_MODES.NORMAL_NPM, then we would get a non premultiplied alpha

Reply:
Thats as far as I understood at least

Reply:
I see. Is this then wrong as well (when using lighting)? https://github.com/jnsmalm/pixi3d/blob/360439e0a7cd94fb52005aa4623a45fbc28334db/src/material/standard/shader/metallic-roughness.frag#L451

Reply:
ah yeah that will have the same issue also

Reply:
May be best making those changes in a separate branch then

Reply:
Maybe not a good idea to change right now because of this: https://github.com/jnsmalm/pixi3d/blob/360439e0a7cd94fb52005aa4623a45fbc28334db/src/gltf/gltf-parser.ts#L217

It will break model loading unfortunately. Have to think a bit about this.

Reply:
Interesting, will have a mess around with this to understand exactly what was meant by opaque alpha being 0